content/blog/getting-started-with-the-hpe-data-services-cloud-console-powershell-sdk.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Getting started with the HPE Data Services Cloud Console Powershell SDK
+date: 2023-08-07T09:01:09.212Z
+author: Thomas Beha
+authorimage: /img/thomasbeha.jpg
+thumbnailimage: /img/dscc-api-spec.png
+disable: false
+tags:
+  - data-services-cloud-console
+  - api
+  - PowerShell
+  - RESTAPI
+  - Storage
+  - hpe-alletra
+---
+The Data Services Cloud Console (DSCC) UI might be sufficient for many customer (maybe even most of them), but lately I was approached from multiple customer that were looking for an efficient way to manage their HPE storage assets with PowerShell scripts. When I started looking into the available option to access the DSCC API with PowerShell I obviously first found the [PowerShell toolkit for managing the HPE Data Services Cloud Console](https://developer.hpe.com/blog/new-powershell-toolkit-available-for-managing-hpe-data-services-cloud-console/) on the [HPE Developer Portal](https://developer.hpe.com/). Working with this PowerShell toolkit I realized that it is on one hand an honorable first approach of my colleagues to write such a toolkit, but it is not complete and not yet fully tested. 
+H﻿ence, a different solution needed to be found.
+
+The DSCC provides an public Open API 3.X spec that can be used to generate your preferred language SDK as it is described for Python in this [blog](https://developer.hpe.com/blog/get-started-building-dscc-api-client-libraries-for-python-using-openapi-generator/). Using the Open API 3.X spec of the DSCC API has the benefit, that it is providing a complete list of possible API operations. And once the procedure to generate the SDK is in place, a new SDK can easily be created if new features are exposed via the DSCC API. 
```

Comment:
The DSCC provides a public OpenAPI 3.X specification that can ...

Second sentence: Using the OpenAPI 3.X specification of the DSCC API ...

I think OpenAPI is spelled in 1 word.

Also, is it OpenAPi 3.X (capital X) or 3.x (lowercase x)? For consistence throughout the blog post, I would go with 3.x.